setup.py

```diff
@@ -17,7 +17,7 @@
 setup(
     name='perfkitbenchmarker',
     url='https://github.com/GoogleCloudPlatform/PerfKitBenchmarker',
-    version='0.6.0',
+    version='0.17.0',
```

Comment:
Will this need to be kept up to date in future releases to avoid tox issues?? If yes, should document that or (preferably) automate it.